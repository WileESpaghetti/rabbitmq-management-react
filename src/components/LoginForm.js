import React, { Component } from 'react';

class LoginForm extends React.Component {
    render() {
        return (
            <div id="login">
                <form>
                    <label>
                        Username
                        <input type="text" name="username" />
                    </label>
                    <label>
                        Password
                        <input type="password" name="password"/>
                    </label>
                    <button type="submit">Login</button>
                </form>
            </div>
        );
    }
}

export default LoginForm;