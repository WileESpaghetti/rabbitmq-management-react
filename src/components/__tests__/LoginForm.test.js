import React from 'react';
import renderer from 'react-test-renderer';
import LoginForm from "../LoginForm";

let wrapper;

describe('<LoginForm />', () => {

    it('renders correctly', () => {
        const tree = renderer
            .create(<LoginForm/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

})



