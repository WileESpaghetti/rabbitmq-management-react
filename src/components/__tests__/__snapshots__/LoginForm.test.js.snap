// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginForm /> renders correctly 1`] = `
<div
  id="login"
>
  <form>
    <label>
      Username
      <input
        name="username"
        type="text"
      />
    </label>
    <label>
      Password
      <input
        name="password"
        type="password"
      />
    </label>
    <button
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
